VoltAir/qml/actors/Orb.qml

```diff
@@ -34,19 +34,22 @@ Actor {
 
     z: 4
 
-    AnimatedImageRenderer {
+    ImageRenderer {
+        function chooseLetter() {
```

Comment:
@diegowald This part will eventually be connected with the issue you are working on (#31), where a dynamic list of letters are imported and stored. Then that list will be randomly used here.